### PR TITLE
#166789984 Deploy App To Heroku 

### PR DESCRIPTION
**Pull request by niyoceles**
#### What does this PR do?
- Deploy the base application to Heroku

#### Description of Task to be completed?
- Setup database on Heroku
- Deploy application to Heroku

#### How should this be manually tested? 
1. Run the following commands in Terminal
```
git clone git@github.com:andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ch-deploy-heroku-166789984
```
2. run `npm run start`

3. Using Postman (or similar) send a `GET` request to [https://apps-92explorers.herokuapp.com/welcome](https://apps-92explorers.herokuapp.com/welcome)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166789984](https://www.pivotaltracker.com/story/show/166789984)
#### Screenshots (if appropriate)
image

#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Comment by kodek-sleuth** [comment]
Title of the PR should have a caps letter at the beginning

**Comment by niyoceles** [comment]
> You are not configuring base app to Heroku but rather deploying the application to heroic and setting up a db, so the description of the tasks to do needs to be changed

Ok, I changed it

**Comment by niyoceles** [comment]
> Some of the files need to have a blank line at the end

Please specify

File: index.js [hunk 1]
```diff
@@ -73,26 +71,29 @@ app.use(function(_req, _res, next) {
...
-
-        res.json({
-            errors: {
-                message: err.message,
-                error: err
-            }
-        });
-    });
-}
+// if (!isProduction) {
```
Comment (dmithamo) [inline index.js]: @niyoceles , do not commit commented-out code.

File: models/databaseUrl.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      pool.query(text, params)
+        .then((res) => {
+          resolve(res);
+        })
+        .catch((err) => {
+          reject(err);
+        });
+    });
+  },
+};
```
Comment (dmithamo) [inline models/databaseUrl.js]: Add a newline at the end of the file.

File: routes/welcomeRouter.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import {Router} from 'express'; 
+import welcomeController from '../controllers/welcomeController';
+
+const router = Router();
+router.get('/', welcomeController.welcome);
+
+export default router;
```
Comment (dmithamo) [inline routes/welcomeRouter.js]: New line required at the eof

File: routes/welcomeRouter.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import {Router} from 'express';
```
Comment (dmithamo) [inline routes/welcomeRouter.js]: Are you sure about this import? Does`Router` exist inside `express`?
COnsider using
```
import express from 'express';
var router = express.Router();
```

**Comment by niyoceles** [comment]
> See the inline comments

Let me do it.

**Comment by niyoceles** [comment]
> > Some of the files need to have a blank line at the end
> 
> Please specify

I corrected them all.

File: routes/welcomeRouter.js (hunk 4 above)
Comment (niyoceles) [reply routes/welcomeRouter.js]: Ok, let me use this `import express from 'express';`

Comment (niyoceles) [reply routes/welcomeRouter.js]: Is better to use this 
`import express from 'express';
const router = express.Router();`
That is correct, thank you.

**Pull request merged**